### PR TITLE
Rename SMALL_SCENERY_FLAG18 to SMALL_SCENERY_FLAG_STACKABLE + some tidying up sub_6E1F34

### PR DESCRIPTION
**Pull request by wolfreak99**

### CODE REVIEW

File: src/openrct2/world/scenery.h [hunk 1]
```diff
@@ -67,7 +67,7 @@ typedef enum {
     SMALL_SCENERY_FLAG15 = (1 << 14),                           // 0x4000
     SMALL_SCENERY_FLAG16 = (1 << 15),                           // 0x8000
     SMALL_SCENERY_FLAG17 = (1 << 16),                           // 0x10000
-    SMALL_SCENERY_FLAG18 = (1 << 17),                           // 0x20000
+    SMALL_SCENERY_FLAG_STACKABLE = (1 << 17),                   // 0x20000
```
Comment (Gymnasiast) [inline src/openrct2/world/scenery.h]: Change comment to `// 0x20000; means scenery item can be placed in the air and over water

**Comment by Gymnasiast** [comment]
Looking good. Left one little line note.

**Comment by wolfreak99** [comment]
I also found that SMALL_SCENERY_FLAG4 defines whether or not to allow manually rotating an object (you know how trees get rotated at random? that's cause SMALL_SCENERY_FLAG4 is disabled)

**Comment by Gymnasiast** [comment]
Yes, that is all right, as they are indeed very related.

**Comment by duncanspumpkin** [comment]
See https://github.com/OpenRCT2/OpenRCT2/pull/5308 this is already captured in my pr

**Comment by Gymnasiast** [comment]
I admit I forgot about that PR. But it has been inactive for quite long. I hope you can finish it soon.
Also, I think that flag 26 has a better name in this PR. Also, this has comments on the flags, whilst yours doesn't. It also refactors some code yours doesn't.

I'd prefer to merge this one and then rebase #5308 on top on it.

**Comment by wolfreak99** [comment]
I feel weird about "I hope you can finish it soon".
@duncanspumpkin If you still have alot to do on your pr, then I feel @Gymnasiast 's plan kinda fits. But if you're near done, and your PR 's code is reasonably up to date, , i'd be more than willing to fork your branch and make a pr with these changes on that and then we could just knock it out through your pr. I'm just trying to provide help where possible.

**Comment by Gymnasiast** [comment]
I was saying that I hope that Duncan finishes his PR soon. Not sure if you interpreted it that way.

**Comment by wolfreak99** [comment]
Yeah I knew that lol.

**Comment by duncanspumpkin** [comment]
I'm not too fussed either way. I made the comment just to make sure there is a link between the two. It is a major pain to refactor my PR when i do get round to finishing it but it will be easier if i see which PR's i need to take reference of.

**Comment by Gymnasiast** [comment]
@wolfreak99 Could you rebase this PR?

**Comment by wolfreak99** [comment]
Want me to go ahead and squash them?

**Pull request merged**